EveHQ.Prism/Classes/StructureBonus.vb

```diff
@@ -0,0 +1,229 @@
+﻿Imports EveHQ.EveData
+
+''' <summary>
+''' A Structure Bonus Definition
+''' </summary>
+Public Class StructureBonus
+
+    Private _group As String
+    Private _materialEfficiency As Decimal
+    Private _timeEfficiency As Decimal
+    Private _costEfficiency As Decimal
+    Private _scienceBonus As Boolean
+
+    ''' <summary>
+    ''' The Group to which the Bonus should be Applied
+    ''' </summary>
+    Public Property Group As String
+        Get
+            Return _group
+        End Get
+        Set(value As String)
+            _group = value
+        End Set
+    End Property
+
+    ''' <summary>
+    ''' The Material Efficiency Rate provided Bonus
+    ''' </summary>
+    Public Property MaterialEfficiency As Decimal
+        Get
+            Return _materialEfficiency
+        End Get
+        Set(value As Decimal)
+            _materialEfficiency = value
+        End Set
+    End Property
+
+    ''' <summary>
+    ''' The Time Efficiency Rate provided Bonus
+    ''' </summary>
+    Public Property TimeEfficiency As Decimal
```

Comment:
It's would be better to separate 2 TE bonuses with 2 properties: ScientificTimeEfficiencyBonus and ManufacturingTimeEfficiencyBonus. Other 2 bonuses: ScientificCostEfficiencyBonus and ManufacturingMaterialEfficiencyBonus. It would be cleaner than the current design with a boolean property IsScientific.